### DISCUSSION TITLE
CRON Job

### DISCUSSION THREAD

**Discussion by mis-uep** [phalcon/cphalcon#15297] [Q&A] [answered]
Hello! How can I create a CRON using Phalcon? Thanks!

**Comment by Jeckerson** [comment, marked as answer]
Hello, via tasks.

Docs: https://docs.phalcon.io/4.0/en/application-cli
Code example: https://github.com/phalcon/rest-api/tree/master/cli

**Reply by mis-uep** [reply]
Thanks! will check on this.

**Reply by quasipickle** [reply]
If this question has been answered, please mark it as so.
